source/GameGraphRoot.cpp

```diff
@@ -290,6 +290,25 @@ bool GameGraphRoot::init(const std::shared_ptr<cugl::AssetManager>& assets,
 		bNode->addChild(buttonLabel);
 	}
 
+	// Overlay Components
+	// Initialize Reconnect Overlay
+	reconnectOverlay = assets->get<Node>("game_overlay_reconnect");
+
+	// Initialize Loss Screen Componenets
+	lossScreen = assets->get<Node>("game_overlay_loss");
+	restartBtn =
+		std::dynamic_pointer_cast<Button>(assets->get<Node>("game_overlay_loss_restartBtn"));
+	levelsBtn = std::dynamic_pointer_cast<Button>(assets->get<Node>("game_overlay_loss_levelsBtn"));
+
+	// Initialize Win Screen Componenets
+	winScreen = assets->get<Node>("game_overlay_win");
+	nextBtn = std::dynamic_pointer_cast<Button>(assets->get<Node>("game_overlay_win_nextBtn"));
+
+	// Register Buttons
+	buttonManager.registerButton(restartBtn);
+	buttonManager.registerButton(levelsBtn);
+	buttonManager.registerButton(nextBtn);
```

Comment:
You have to call `buttonManager.process()` during the update loop to actually make the buttons go up and down; otherwise the button manager doesn't do anything.